tests/test_randwalk.py

```diff
@@ -0,0 +1,165 @@
+# Copyright (c) 2012-2016 by the GalSim developers team on GitHub
+# https://github.com/GalSim-developers
+#
+# This file is part of GalSim: The modular galaxy image simulation toolkit.
+# https://github.com/GalSim-developers/GalSim
+#
+# GalSim is free software: redistribution and use in source and binary forms,
+# with or without modification, are permitted provided that the following
+# conditions are met:
+#
+# 1. Redistributions of source code must retain the above copyright notice, this
+#    list of conditions, and the disclaimer given in the accompanying LICENSE
+#    file.
+# 2. Redistributions in binary form must reproduce the above copyright notice,
+#    this list of conditions, and the disclaimer given in the documentation
+#    and/or other materials provided with the distribution.
+#
+
+from __future__ import print_function
+import numpy
+import os
+import sys
+import galsim
+
+from galsim_test_helpers import *
+
+try:
+    import galsim
+except ImportError:
+    sys.path.append(os.path.abspath(os.path.join(path, "..")))
+    import galsim
+
+
+@timer
+def test_randwalk_defaults():
+    """
+    Create two objects, each with three exposures. Save them to a MEDS file.
+    Load the MEDS file. Compare the created objects with the one read by MEDS.
+    """
+
+    # try constructing with mostly defaults
+    npoints=100
+    hlr = 8.0
+    rw=galsim.RandomWalk(npoints, hlr)
+
+    assert rw.npoints==npoints,"expected npoints==%d" % (npoints, rw.npoints)
+    assert rw.half_light_radius==hlr,\
+        "expected hlr==%g, got %g" % (hlr, rw.half_light_radius)
+
+    g=rw.gaussians
+    ngauss=len(g)
+    assert ngauss == npoints==npoints,"expected %d gaussians, got %d" % (npoints, ngauss)
+
+@timer
+def test_randwalk_valid_inputs():
+    """
+    Create two objects, each with three exposures. Save them to a MEDS file.
+    Load the MEDS file. Compare the created objects with the one read by MEDS.
+    """
```

Comment:
This comment doesn't seem to apply to the below tests.  (Likewise for the other test functions.)